### ISSUE TITLE
Single-cell and bulk transcriptome sequencing identifies two epithelial tumor cell states and refines the consensus molecular classification of colorectal cancer

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#27]
## TL;DR

The consensus molecular subtype (CMS) classification of colorectal cancer is based on bulk transcriptomics. The underlying epithelial cell diversity remains unclear. We analyzed 373,058 single-cell transcriptomes from 63 patients, focusing on 49,155 epithelial cells. We identified a pervasive genetic and transcriptomic dichotomy of malignant cells, based on distinct gene expression, DNA copy number and gene regulatory network. We recapitulated these subtypes in bulk transcriptomes from 3,614 patients.

### Paper Link

https://www.nature.com/articles/s41588-022-01100-4

### Author/Institution

Ignasius Joanito (Genome Institute of Singapore)

## Methods

colorectal cancer. survival analysis. network analysis.

## Cite

Joanito, I., Wirapati, P., Zhao, N. et al. Single-cell and bulk transcriptome sequencing identifies two epithelial tumor cell states and refines the consensus molecular classification of colorectal cancer. Nat Genet 54, 963–975 (2022). https://doi.org/10.1038/s41588-022-01100-4
